### PR TITLE
[11.0] [MIG] web_widget_color

### PR DESCRIPTION
**Pull request by etobella**
Migration to 11.0
As it was commented on #775

### CODE REVIEW

File: web_widget_color/README.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  |listview|
+
+
+Usage
+=====
+
+You need to declare a char field of at least size 7::
+
+    _columns = {
```
Comment (katyukha) [inline web_widget_color/README.rst]: It seems that we do not need 7.0 API example in README for 11.0 version of addon.

**Review by rgarnau — APPROVED** [review]
Functional review :+1:

File: web_widget_color/README.rst (hunk 1 above)
Comment (etobella) [reply web_widget_color/README.rst]: True, it has been removed

File: web_widget_color/README.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    )
+
+
+In the view declaration, put widget='color' attribute in the field tag::
+
+    ...
+    <field name="arch" type="xml">
+        <tree string="View name">
+            ...
+            <field name="name"/>
```
Comment (pedrobaeza) [inline web_widget_color/README.rst]: Remove this useless line that can confuse also. Is it also valid for form. Then it should be mentioned.

File: web_widget_color/README.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+Credits
+=======
+
+Contributors
+------------
+
+* Adil Houmadi <adil.houmadi@gmail.com>
+* Nicolas JEUDY (Sudokeys) <https://www.github.com/njeudy>
+
```
Comment (pedrobaeza) [inline web_widget_color/README.rst]: Add yourself as contributor

File: setup/web_widget_color/odoo/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```
Comment (sbidoul) [inline setup/web_widget_color/odoo/__init__.py]: for 11.0, remove this `__init__.py`

File: setup/web_widget_color/odoo/addons/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```
Comment (sbidoul) [inline setup/web_widget_color/odoo/addons/__init__.py]: for 11.0, remove this `__init__.py`

**Review by HviorForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**